### PR TITLE
Fix Lucene SDK issues

### PR DESCRIPTION
**Pull request by Noah-ney**
update pim-lucene depending on the new version of SDK

### CODE REVIEW

File: lucene/pim-benchmarks/exact_phrase/Makefile [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+INDEX_DIRECTORY:=/scratch/nney/lucene_data
 DATASET_PATH:=./datasets/wiki_subset
 #DATASET_PATH:=/home/shared/index_search/wikipedia/files
 #OUTPUT_PREFIX:=full_wiki
 #OUTPUT_PREFIX:=superset
 OUTPUT_PREFIX:=subset
 QUERIES:=./queries/requests.txt
 #QUERIES:=./queries/requests_one_phrase.txt
-NB_DPUS=2048
+NB_DPUS=1024
```
Comment (yberthier) [inline lucene/pim-benchmarks/exact_phrase/Makefile]: Let 2048

File: lucene/pim-benchmarks/exact_phrase/Makefile
Comment (yberthier) [inline lucene/pim-benchmarks/exact_phrase/Makefile:1]: Let as before

**Review by yberthier — CHANGES_REQUESTED** [review]
In our use cases we have to run Lucene over different number of DPUs.

It could be better to set NR_DPUS as environnement variable instead of touching the source code.

**Review by SylvanBrocard — CHANGES_REQUESTED** [review]
lgtm
Just add a mention of that environment variable in the README.

**Review by SylvanBrocard — APPROVED** [review]

**Pull request merged**